pkg/apis/servicecatalog/doc.go

```diff
@@ -19,5 +19,5 @@ limitations under the License.
 
 // Package servicecatalog contains all of the sources needed to make
 // servicebrokers and their related service objects.
-// +groupName=
+// +groupName=servicecatalog.k8s.io
```

Comment:
in k8s, the group name can be empty for the legacy api group.  i would like to figure out a way in the machinery to throw a warning of some kind.  it may for example make sense to have to add a "legacy" annotation that allows for the empty group name, but absent that annotation, the generator could just fail.